services/bettersocial/fcmToken/sendChatNotificationByChannelMembers.js

```diff
@@ -12,7 +12,7 @@ const sendChatNotificationByChannelMembers = async (
       ...payload,
       data: {
         ...payload.data,
-        is_annoymous: member.is_anonymous.toString()
+        is_annoymous: member?.is_anonymous?.toString() || 'false'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a typo in the property name `is_annoymous`. It should be `is_anonymous` to match the context of the PR title and the old hunk.
```diff
-         is_annoymous: member?.is_anonymous?.toString() || 'false'
+         is_anonymous: member?.is_anonymous?.toString() || 'false'
```
Also, it's good that you're using optional chaining (`?.`) to prevent potential null reference errors. However, keep in mind that if `member` or `is_anonymous` is undefined, the value will default to 'false'. Make sure this is the intended behavior.


<!-- This is an auto-generated comment by OSS CodeRabbit -->